catalog/spatial/wfs/1.0.0/spatial-wfs-v1_0_0-schema-webapp/pom.xml

```diff
@@ -42,9 +33,6 @@
                         <Web-ContextPath>/ogc</Web-ContextPath>
                         <_wab>src/main/resources</_wab>
                         <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
-                        <Embed-Dependency>
```

Comment:
:question: If you don't embed jetty-servlets, wouldn't you need to import the packages exported by the jetty-servlets bundle?

Reply:
The artifact is already running by default and several other bundles are already importing the packages.